pandas/core/strings.py

```diff
@@ -413,58 +413,12 @@ def _get_single_group_name(rx):
         return None
 
 
-def str_extract(arr, pat, flags=0):
+def str_extract_original(arr, pat, flags=0):
```

Comment:
I would make this and ``str_extract_frame`` both private, e.g. ``_str_extract_frame``